package.json

```diff
@@ -37,7 +37,7 @@
     "js-yaml": "^3.14.0"
   },
   "dependencies": {
-    "@startbahn/startrail": "1.8.0-beta-3",
+    "@startbahn/startrail": "git+ssh://git@github.com:startbahn/startrail#a9477ce2ef0cd518c5deb1ed2e6785ede97b3679",
```

Comment:
@ardhiyanyanu looks the CI build is failing because of this line. i guess CI is not setup to refer to it like this. can we point at a version that has been released to npm instead? whatever the latest develop is pointing to should be fine ..

Reply:
right now if I use released version of startrail, it will fail in build because startrail have 
```
"engines": {
    "node": ">=14.16 <14.17"
},
```

Reply:
so the plan is to change it after startrail is released with bump up changes